ui/DesignSystem/Component/EmptyState.svelte

```diff
@@ -0,0 +1,60 @@
+<script>
+  import Icon from "./EmptyState/Icon.svelte";
+  import { Button, Text } from "../Primitive";
+
+  export let icon = "tent";
+  export let text = "Nothing to see here";
+  export let mainCtaText = "Go back";
```

Comment:
My preference is always to spell things out, I find it reduces confusion when reading code.

Could we maybe call the props `primaryActionText` and `secondaryActionText`?

Reply:
addressed in (ed0bd95)[https://github.com/radicle-dev/radicle-upstream/pull/537/commits/ed0bd952a76832c19aa7cdd6c8d6f783d0cc78d3]